packages/pds/src/app-view/db/tables/notification-push-token.ts

```diff
@@ -0,0 +1,11 @@
+export const tableName = 'notification_push_token'
```

Comment:
we don't need to store these on the pds either i think

Reply:
Yup, I'll remove

Reply:
removed.